### PR TITLE
update variant legend group ids

### PR DESCRIPTION
**Pull request by jyothishnt**
## Description
This PR updates variant group ids based on its values in the data file.

## Related JIRA Issue(s)
https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-1890

## Deployment URL(s)
http://update-var-group.review.ensembl.org

## Views affected
GB

### CODE REVIEW

File: src/content/app/genome-browser/components/drawer/drawer-views/variant-group-legend/VariantGroupLegend.scss [hunk 1]
```diff
@@ -31,19 +31,19 @@
 }
 
 .variantColour1 {
-  background-color: $duckegg-blue;
+  background-color: $dark-pink;
```
Comment (jyothishnt) [inline src/content/app/genome-browser/components/drawer/drawer-views/variant-group-legend/VariantGroupLegend.scss:34]: git diff is bit weird. I only changed the class name and order of these :-)

**Review by azangru — APPROVED** [review]
Should merge together with updated genome browser backend.

**Comment by jyothishnt** [comment]
> Should merge together with updated genome browser backend.

Why? This change doesn't have any effect as we dont use ids anywhere yet.

**Comment by azangru** [comment]
> Why? This change doesn't have any effect as we dont use ids anywhere yet other than the linking to classnames.

True. Feel free to merge then.

**Pull request merged**